engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -1342,6 +1343,19 @@ class demon_hunter_action_t : public Base
     ab::apply_affecting_aura( p->talent.havoc.any_means_necessary );
     ab::apply_affecting_aura( p->talent.havoc.dancing_with_fate );
 
+    ab::apply_affecting_aura( p->talent.vengeance.perfectly_balanced_glaive );
+    ab::apply_affecting_aura( p->talent.vengeance.meteoric_strikes );
+    ab::apply_affecting_aura( p->talent.vengeance.shear_fury );
```

Comment:
Unfortunately since this is done as Spell Effect 2 indirection rather than a mod aura, you'll have to add this manually into the Shear energize_amount in the shear_t() constructor.